README.md

```diff
@@ -73,6 +73,42 @@ logger.info({
 })
 ```
 
+## Custom Log Levels
+
+As per the Winston documentation, [custom log levels](https://github.com/winstonjs/winston#using-custom-logging-levels) may be used. In order to use such
+levels in LogDNA, [custom levels must be defined](https://github.com/logdna/logger-node#custom-log-levels) for that logger as well.
+
+```javascript
+  const levels = {
+    error: 0
+  , warn: 1
+  , info: 2
+  , http: 3
+  , verbose: 4
+  , loquatious: 5
+  , ludicrous: 6
+  }
```

Comment:
Do you think it'd be worthwhile to offer a helper function for generating levels paired with an index? I know it's a Winston specific thing, but it can be a bit annoying having to maintain these parities, especially when you add/remove to the beginning/middle.

```javascript
const levels = logdnaWinston.buildLevels([
  'silent'
, 'error'
, 'warn'
, 'info'
, 'http'
, 'verbose'
, 'loquacious'
, 'ludicrous'
])

const logger = winston.createLogger({
  levels
, level: 'ludicrous'
})
const logdna_options = {
  key: 'abc123'
, levels: levels.toLogDNA()
}
// this is just an example, a different api might be better
```

it's such a small thing, just throwing it out there

Reply:
That feels a bit overly abstracted to me at this point since we don't really have feedback on whether that's useful.  For me, doing an `Object.keys(winston_levels)` doesn't get much easier and it prevents misdirection.  Let's go with the simple solution now, especially since these are instantiation things.  It should be 1 spot in the code, you know?